### PR TITLE
add qti-haptics vibrator control support

### PR DESCRIPTION
**Pull request by areyoudeveloper**

### CODE REVIEW

**Comment by areyoudeveloper** [comment]
idk how to add it correcry please fix it ,
its worked on https://github.com/areyoudeveloper/SmartPack-Kernel-Manager/commit/8b2309ae555d885dd41c4d8cad2007fcc91632db

**Comment by areyoudeveloper** [comment]
another ref https://github.com/areyoudeveloper/android_kernel_xiaomi_msm8937-2/commit/8f3ef977894151c09c7eabf91b9c2f1f577a5f4d

File: app/src/main/java/com/smartpack/kernelmanager/utils/kernel/misc/qtihaptics.java [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+        if (sInstance == null) {
+            sInstance = new Vibration();
+        }
+        return sInstance;
+    }
+
+    private static final String VIB_LIGHT = "/sys/module/qti_haptics/parameters/vmax_mv_override";
+    private static final String VIB_ENABLE = "/sys/module/qti_haptics/parameters/vmax_mv_enable";
+
+    private final HashMap<String, MinMax> mVibrations = new HashMap<>();
```
Comment (Lennoard) [inline app/src/main/java/com/smartpack/kernelmanager/utils/kernel/misc/qtihaptics.java:48]: For the implementation, I believe the other entries must be preserved on the list.

File: app/src/main/java/com/smartpack/kernelmanager/utils/kernel/misc/qtihaptics.java [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+    public static Vibration getInstance() {
+        if (sInstance == null) {
+            sInstance = new Vibration();
+        }
+        return sInstance;
+    }
+
+    private static final String VIB_LIGHT = "/sys/module/qti_haptics/parameters/vmax_mv_override";
+    private static final String VIB_ENABLE = "/sys/module/qti_haptics/parameters/vmax_mv_enable";
```
Comment (Lennoard) [inline app/src/main/java/com/smartpack/kernelmanager/utils/kernel/misc/qtihaptics.java:46]: Old constants should be preserved. I believe we can still implement it in Vibration.java.

**Comment by sunilpaulmathew** [comment]
@areyoudeveloper @Lennoard 
I'm sorry, I was in holidays & busy with personal reasons. I'll take are of this soon. Thanks

**Comment by sunilpaulmathew** [comment]
@areyoudeveloper 
Are you available to test? If so, please get the test APK from my [Dropbox](https://www.dropbox.com/s/o8v6s7wal7yp007/com.smartpack.kernelmanager.apk?dl=0). In case the link expires, please let me know. Thanks

**Comment by areyoudeveloper** [comment]
> @areyoudeveloper Are you available to test? If so, please get the test APK from my [Dropbox](https://www.dropbox.com/s/o8v6s7wal7yp007/com.smartpack.kernelmanager.apk?dl=0). In case the link expires, please let me know. Thanks

It's works 🥳

**Pull request closed without merging**